airflow-core/src/airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -75,6 +75,7 @@ class TITerminalStatePayload(StrictBaseModel):
 
     end_date: UtcDateTime
     """When the task completed executing"""
+    rendered_map_index: Annotated[str | None, Field(default_factory=str)]
```

Comment:
Shouldn't these all default to none?

```suggestion
    rendered_map_index: str | None = None
```

So that we can tell the difference between not passed and empty string?